### DISCUSSION TITLE
btcpay.config

### DISCUSSION THREAD

**Discussion by SireWilliamson** [btcpayserver/btcpayserver#2199] [Technical Support] [answered]
Where can I find the documentation on the options for the btcpay.config file?
I read in the "manual" section for install that you could use MySQL, but I can't find what I need to set in the btcpay.config to do that.
Thank you.

**Comment by janoside** [comment, marked as answer]
Referencing [this line of code](https://github.com/btcpayserver/btcpayserver/blob/600cc2042399a84bc7eddfcadfe5729e4edab95b/BTCPayServer/Configuration/DefaultConfiguration.cs#L126), which produces the default configuration file, I believe you need to include a line like:

    mysql=User ID=root;Password=myPassword;Host=localhost;Port=3306;Database=myDataBase;

**Reply by SireWilliamson** [reply]
Thank you, that is what I needed to know.

**Comment by Kukks** [comment]
I would highly discourage the usage of the mysql version

**Reply by SireWilliamson** [reply]
Is it a performance issue?

**Reply by bolatovumar** [reply]
I believe PostgreSQL is what BTCPay is primarily built in mind with.

**Comment by janoside** [comment]
@Kukks Thanks - that's good to know. Further explanation would probably be helpful to the original poster, plus I'm just curious to understand your reasoning. Is this along the same lines as strenuously discouraging any non-docker deployment? Is the mysql configuration unsupported/deprecated/broken and removal is just not a priority?
